apps/condo/domains/property/utils/testSchema/factories.js

```diff
@@ -96,7 +97,65 @@ const buildFakeAddressAndMeta = (withFlat, addressMetaExtraAttrs = {}) => {
     return { address, addressMeta }
 }
 
+const buildPropertyMap = ({ floors = 3, unitsOnFloor = 4, parkingFloors = 2, unitsOnParkingFloor = 8 } = {}) => ({
```

Comment:
You can also build n sections here

Reply:
Looks like it is not necessary to build sections similar to each other. We need just the fact the flat in the building. Not matter which section it in.